spot_description/urdf/accessories.urdf.xacro

```diff
@@ -77,4 +77,49 @@
       </joint>
     </xacro:if>
   </xacro:if>
+
+  <xacro:if value="$(optenv CHAMP_GAZEBO 0)">
+    <xacro:include filename="$(find spot_description)/urdf/constants.urdf.xacro" />
+    <xacro:include filename="$(find hector_sensors_description)/urdf/hokuyo_utm30lx.urdf.xacro" />
+    <xacro:include filename="$(find hector_sensors_description)/urdf/asus_camera.urdf.xacro" />
+
+    <xacro:hokuyo_utm30lx name="hokuyo" parent="base_link" ros_topic="scan" update_rate="30" ray_count="1040" min_angle="130" max_angle="-130">
+      <origin xyz="0.0 0.0 ${base_height / 2 + 0.04}" rpy="0 0 0"/>
+    </xacro:hokuyo_utm30lx>
+
+    <gazebo reference="hokuyo_frame">
+      <material>Gazebo/FlatBlack</material>
+    </gazebo>
+
+    <xacro:asus_camera parent="base_link" name="camera">
+      <origin xyz="${base_length / 2 - 0.05} 0.0 -${base_height / 2 + 0.01}" rpy="0 0 0"/>
+    </xacro:asus_camera>
```

Comment:
Are these appropriate models for Spot's cameras/LIDAR? For example, I think Velodyne might be more accurate.